src/main/java/seedu/taassist/logic/commands/ClearCommand.java

```diff
@@ -13,11 +13,13 @@ public class ClearCommand extends Command {
     public static final String COMMAND_WORD = "clear";
     public static final String MESSAGE_SUCCESS = "TA-Assist has been cleared!";
 
-
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+        if (model.isInFocusMode()) {
```

Comment:
To not break the abstraction barrier as much, would it be better for this to focusMode check to be handled by `Model::setTaAssist` instead? Then `ModelManager`'s test should be tested instead of `ClearCommand`'s